packages/ui-components/src/components/button.tsx

```diff
@@ -28,18 +33,13 @@ export interface IButtonProps
  * @returns Component
  */
 export function Button(props: IButtonProps): JSX.Element {
```

Comment:
From the discussion with the others dev to limit the impact on extensions, we won't modify this component. Could you please undo the changes in this file? Instead in the doc string of `Button` add a deprecation notice like:

```ts
@deprecated You should use the `Button` component from `@jupyter/react-components` instead.
```